packages/db/index.js

```diff
@@ -1,59 +1,11 @@
 import { resolve, validationOptions } from '@platformatic/basic'
 import { kMetadata, loadConfiguration as utilsLoadConfiguration } from '@platformatic/foundation'
-import { transform as serviceTransform } from '@platformatic/service'
-import { readFile } from 'node:fs/promises'
-import { resolve as resolvePath } from 'node:path'
 import { DatabaseCapability } from './lib/capability.js'
 import { schema } from './lib/schema.js'
 import { upgrade } from './lib/upgrade.js'
+import { transform as _transform } from './lib/config-transform.js'
 
-export async function transform (config) {
-  config = await serviceTransform(config)
-
-  if (
-    config.db &&
-    config.db.connectionString.indexOf('sqlite') === 0 &&
-    config.db.connectionString !== 'sqlite://:memory:'
-  ) {
-    const originalSqlitePath = config.db.connectionString.replace('sqlite://', '')
-    const sqliteFullPath = resolvePath(config[kMetadata].root, originalSqlitePath)
-    config.db.connectionString = 'sqlite://' + sqliteFullPath
-  }
-
-  /* c8 ignore next 3 */
-  if (config.db.graphql?.schemaPath) {
-    config.db.graphql.schema = await readFile(config.db.graphql.schemaPath, 'utf8')
-  }
-
-  /* c8 ignore next 2 */
-  const arePostgresqlSchemaDefined =
-    config.db?.connectionString.indexOf('postgres') === 0 && config.db?.schema?.length > 0
-  const migrationsTableName = arePostgresqlSchemaDefined ? 'public.versions' : 'versions'
-
-  // relative-to-absolute migrations path
-  if (config.migrations) {
-    config.migrations.table = config.migrations.table || migrationsTableName
-  }
-
-  if (config.migrations && config.db) {
-    // TODO remove the ignores
-    /* c8 ignore next 4 */
-    config.db.ignore = config.db.ignore || {}
-    config.db.ignore = Object.assign(
-      {},
-      {
-        [config.migrations.table || migrationsTableName]: true
-      },
-      config.db.ignore
-    )
-  }
-
-  if (config.types?.autogenerate === 'true') {
-    config.types.autogenerate = true
-  }
-
-  return config
-}
+export const transform = _transform
```

Comment:
```suggestion
export { transform } from './lib/config-transform.js'
```

Reply:
This is not possible because it's used inside.